hw/ip/pattgen/doc/checklist.md

```diff
@@ -163,18 +163,18 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
 Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | N/A         |
-Documentation | [DV_DOC_COMPLETED][]                    | Not Started |
+Documentation | [DV_DOC_COMPLETED][]                    | Done        |
```

Comment:
this item is not complete either - the doc still has contains template text.

Reply:
Fix it